website/content/docs/integrations/vault/acl.mdx

```diff
@@ -140,14 +260,66 @@ path "secret/metadata/*" {
 }
 ```
 
-Using that policy configuration file, create the Vault policy.
+</CodeBlockConfig>
 
-```shell-session
-$ vault policy write nomad-workloads vault-policy.hcl
-```
+The overall configuration structure is illustrated in the following diagram.
+
+[![Vault Auth Overview][img_vault_auth_overview]][img_vault_auth_overview]
+
+#### Vault Namespaces <EnterpriseAlert inline />
+
+Vault Enterprise supports multiple namespaces and jobs in Nomad Enterprise can
+use the [`vault.namespace`][] parameter to specify which namespace to use. In a
+multi-namespace environment, the authentication setup described must be applied
+to each Vault namespace used by jobs.
 
-You can write additional Vault policies that give Nomad workloads using the
-default role access to other secrets.
+### Important Considerations About the JWKS URL
+
+The recommended configuration assumes Vault servers are able to connect to
+Nomad agents (either client or servers) to retrieve the JSON Web Key Set
+information.
+
+This section covers additional aspects you should consider depending on how
+your Vault and Nomad clusters are configured and deployed.
+
+#### Mutual TLS in Nomad
+
+It is highly recommended to use [mutual TLS][tutorial_mtls] in production
+deployments of Nomad. With mTLS enabled, the [`tls.verify_https_client`][]
+configuration must be set to `false` since it is not possible to provide client
+certificates to the Vault auth method.
+
+Alternatively, you may expose Nomad's JWKS URL from a proxy or a load balancer
+that handles the mutual TLS connection to Nomad and exposes the JWKS URL
+endpoint over standard TLS.
```

Comment:
Is there a name for non-mutual TLS? I used "standard TLS" but I'm not sure if that's right 😅